server/main.jai

```diff
@@ -397,14 +399,18 @@ handle_request :: (request: LSP_Request_Message, raw_request: string) {
 main :: () {
     context.logger = lsp_log;
 
-    success: bool;
-    success, server.args = parse_arguments(Args);
+    success, args := parse_arguments(Args);
     if !success {
         log_error("Could not parse arguments");
         exit(1);
     }
+    log("Args are %", args);
```

Comment:
subtle difference in logging here. Previously the code logged the args including the result of any compiler path finding work, now it does not, which seems fine as a) more honest w.r.t. actual passed args & b) the compiler path finding has its own logging.